packages/x-components/tests/e2e/cucumber/no-suggestions.feature

```diff
@@ -18,5 +18,5 @@ Feature: No Suggestions component
 
     Examples:
       | queryWithoutSuggestions | queryWithSuggestions |
-      | asdfg                   | puzzle               |
+      | asdfg                   | lego                 |
```

Comment:
I think this one is pending and also I would rename it to no-results as suggestions is confused.

There are not results no suggestions.

Reply:
This test was not mocked. I don't know why. The suggestions adapter can be mocked, and is mocked in other places. Again, I would remove it in another PR explaining the breaking change. [Task here](https://searchbroker.atlassian.net/browse/EX-4735)

Reply:
Check the component and test again, it is talking about having no suggestions, results are not involved at all here. In any case it will be removed. This was made specifically for kroger and they never used it.